experiment/secret-sync-controller/README.md

```diff
@@ -0,0 +1,71 @@
+# secret_sync
+Synchronizing between Google Cloud Secret Manager secrets and Kubernetes secrets.
+
+## Current Functions
+Parse configuration file into source and destination secrets.
+
+Fetch the latest versions of from Secret Manager secret and Kubernetes secrets.
+
+## Prerequisites
+- Create a gke cluster.
+
+- [Create service account for app](https://cloud.google.com/docs/authentication/production#command-line)
+
+- Grant required permissions to the service account `service-account-name`.
+
+	- Permission to get clusters:
+
+		    gcloud projects add-iam-policy-binding k8s-jkns-gke-soak --member "serviceAccount:<service-account-name>@k8s-jkns-gke-soak.iam.gserviceaccount.com" --role "roles/container.clusterViewer"
```

Comment:
nit: replace occurrences of k8s-jkns-gke-soak with <project-id>

Better yet could be replacing this with a script that runs all of these commands with the appropriate parameters

Reply:
Replaced.
Working on a script containing all these, as well as setting up workload identity.